dev-docs/source/Testing/Utility/FilterEventsInterfaceTest.rst

```diff
@@ -0,0 +1,60 @@
+.. _filter_event_testing:
+
+Filter Events Interface Testing
+===============================
+
+.. contents::
+   :local:
+
+*Prerequisites*
+
+- Download the `Usage Examples <http://download.mantidproject.org>`_
+
+*Note: At the time of writing this test, there's an issue pending with data validation on this interface(#36073); if this issue hasn't been
+corrected yet, please remind the team about it, else update documentation and remove this note.*
+
+Filter Events
+--------------
+
+
+**Time required: 10-15 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``Utility`` > ``Filter Events``
+#. Browse for file 'CNCS_7860_Event.nxs' from the Usage Data set.
+#. Click ``Load``. After few seconds, plot should be updated with a graph of summed up Counts vs.Time.
+#. ``_Summed_..`` workspace should be generated on the ADS, as well as ``CNCS_7860_Event`` workspace.
+#. Clicking on ``Refresh`` button should update the drop-down list with the name of the loaded workspace.
+#. Check Vertical Range Sliders work properly. You can't overlap the sliders or cross the sliders.
+   Additionally, placing lower slider at the bottom of the range allows the other slider to move freely on the whole range.
+#. Check Horizontal Range Sliders work properly. You can't overlap or cross the sliders.
+   Additionally, having left slider at the leftmost part of the range allows the other slider to move on the whole range.
+#. Check that the Text Edits for ``Starting Time`` and ``Stopping Time`` work correctly:
+
+   - Position of vertical bars on the plot are updated appropiately when changing the numerical value of the edits and clicking ``Set``.
+   - Setting a value on ``Starting Time`` larger than the current value on ``Stopping Time`` is not allowed, and viceversa.
```

Comment:
Yeah, it doesn't happen with every pair of values. so it may be some kind of rounding error. I will open an issue, there are many issues with this interface. I plan to work on it on next release.

Reply:
Sounds like a good idea 👍